central/main.go

```diff
@@ -239,6 +240,8 @@ func main() {
 		return
 	}
 
+	clientconn.SetUserAgent("central")
```

Comment:
does it make sense to put more data in user agent like version and platform like Mozilla do https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent/Firefox